packages-exp/app-check-exp/src/internal-api.ts

```diff
@@ -292,8 +293,12 @@ function notifyTokenListeners(
           observer.next(token);
         }
       } else {
-        // Otherwise return the token, whether or not it has an error field.
-        observer.next(token);
+        // If this is a 2P listener, return the token, whether or not it
+        // has an error field. If it is a 3P listener with no error handler,
+        // ignore the error.
+        if (observer.type === ListenerType['2P']) {
```

Comment:
Can it be just `ListenerType.2P`?

Reply:
No, syntax error if it starts with a number and isn't in a string.

Reply:
Can you rename the enum members to start with a letter instead, perhaps `Internal` vs `External`? The indexing style is inconsistent with our code.

Reply:
Renamed to `INTERNAL`/`EXTERNAL`.